src/components/stacking.tsx

```diff
@@ -57,7 +57,10 @@ export const StackingPercentage = ({ balances, stackingBlock }: any) => {
                 </Title>
               </Flex>
               <Box textAlign="center">
-                <Caption mb="base-tight">Stacking for ~{blocksUntilUnlocked} more blocks</Caption>
+                <Caption mb="base-tight">
+                  ~{blocksUntilUnlocked.toLocaleString(undefined, { maximumFractionDigits: 2 })}{' '}
```

Comment:
I had issues using this in wallet, doesn't handle big (unlikely but theoretically possible) numbers